src/injector.ts

```diff
@@ -190,58 +352,86 @@ export default class Injector {
    * @param  {string}            chain      chain name (ex: 'ropsten')
    * @param  {string}            address    contract address
    * @param  {string[]}          files
-   * @return {Promise<string[]>}            addresses of successfully verified contracts
+   * @return {Promise<void>}
```

Comment:
Forgot to update docs comment here...
```suggestion
   * @return {Promise<string[]>}            addresses of successfully verified contracts
```